### PR TITLE
Change proxy to istio-proxy on tasks description

### PR DESCRIPTION
**Pull request by ayj**
Fixes https://github.com/istio/issues/issues/70

### CODE REVIEW

File: _docs/setup/kubernetes/manual-sidecar-inject.md [hunk 1]
```diff
@@ -181,17 +201,23 @@ applications which are redirected to proxy.
           apiVersion: v1
           fieldPath: metadata.namespace
     -
-      name: POD_IP
+      name: INSTANCE_IP
```
Comment (andraxylia) [inline _docs/setup/kubernetes/manual-sidecar-inject.md]: in the future we can rename this to POD_IP, it seems to be an unwritten standard.

File: _docs/setup/kubernetes/manual-sidecar-inject.md [hunk 2]
```diff
@@ -204,20 +230,22 @@ traffic to the proxy. An init-container is used for two reasons:
...
-     }
-   },
+   ```yaml
+      initContainers:
+      - args:
+        - -p
+        - "15001"
+        - -u
+        - "1337"
+        image: docker.io/istio/proxy_init:unittest
```
Comment (andraxylia) [inline _docs/setup/kubernetes/manual-sidecar-inject.md]: unittest?

File: _docs/setup/kubernetes/manual-sidecar-inject.md (hunk 1 above)
Comment (ayj) [reply _docs/setup/kubernetes/manual-sidecar-inject.md]: Changed to reflect the actual value used [here](https://github.com/istio/pilot/blob/master/platform/kube/inject/inject.go#L426)

**Pull request merged**